src/tools/tidy/src/features.rs

```diff
@@ -166,8 +166,7 @@ pub fn check(path: &Path, bad: &mut bool) {
 
     // FIXME get this whitelist empty.
     let whitelist = vec![
-        "abi_ptx", "simd", "macro_reexport",
-        "static_recursion", "quote",
+        "abi_ptx", "simd", "static_recursion", "reflect",
```

Comment:
It was removed by https://github.com/rust-lang/rust/pull/39075, so shouldn't be present.